fakeredis/commands_mixins/scripting_mixin.py

```diff
@@ -68,6 +73,51 @@ def _lua_redis_log(lua_runtime: LUA_MODULE.LuaRuntime, expected_globals: Set[Any
     LOGGER.log(REDIS_LOG_LEVELS_TO_LOGGING[lvl], msg)
 
 
+_lua_cjson_null = object()  # sentinel value
+
+
+def _cjson_python_to_lua(obj: Any) -> Any:
+    """Convert a pure python object obtained after JSON deserialization into a usable object in the lua runtime."""
+    if obj is None:
+        return _lua_cjson_null
+    if isinstance(obj, str):
+        return obj.encode()
+    if isinstance(obj, list):
+        return [_cjson_python_to_lua(item) for item in obj]
+    if isinstance(obj, dict):
+        return {_cjson_python_to_lua(key): _cjson_python_to_lua(value) for key, value in obj.items()}
+    return obj
+
+
+def _cjson_lua_to_python(obj: Any) -> Any:
+    """Convert a passed lua runtime object obtained before JSON serialization into a pure python object."""
+    if obj is _lua_cjson_null:
+        return None
+    if isinstance(obj, bytes):
+        return obj.decode()
+
+    lua_type = LUA_MODULE.lua_type(obj)
+    if lua_type == "table":
+        d = dict(obj)  # TODO: This is very naive, lua tables aren't just dicts
```

Comment:
I'm aware that this probably needs a better solution, however, I'm not sure how to handle it. I'm open for any suggestions.

Reply:
Fixed in [856efa6](https://github.com/cunla/fakeredis-py/pull/399/commits/856efa685ac0812570adc22bb0fe5ce102650b84)